content/testing-yaml/testing.md

```diff
@@ -21,37 +23,66 @@ npm test
 For non-UI tests or unit testing:
 
 ```bash
-./gradlew test
+- name: Test
+  script: ./gradlew test
+  test_report: app/build/test-results/**/*.xml
 ```
 
 UI tests (also known as instrumented tests):
 
 ```bash
-./gradlew connectedAndroidTest
+- name: Launch emulator
+  script: |
+    cd $ANDROID_HOME/tools
+    emulator -avd emulator &
+    adb wait-for-device
+- name: Test
+  script: |
+    set -e
+    ./gradlew connectedAndroidTest
```

Comment:
Doesn't `./gradlew connectedAndroidTest` produce xml report?

Reply:
added in 2e33bafc3592fa7408d75ac9459306eb97d56a8d